src/AppStream.Validation.Plugin/appsettings.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "AIPlugin": {
+    "SchemaVersion": "v1",
+    "NameForModel": "JSON or XML validator",
+    "NameForHuman": "JSON or XML validator",
+    "DescriptionForModel": "Help the user validate JSONs or XMLs. You can check whether a given string is a valid JSON or XML.",
+    "DescriptionForHuman": "[ChatGPT](https://openai.com/blog/chatgpt-plugins)/[Semantic Kernel](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/plugins?tabs=Csharp) plugin for validating JSONs or XMLs.",
```

Comment:
Maybe rather:
`Help the user validate JSON or XML object. You can validate format or validate it agianst provided schema or provided sample object`
?